src/md/winmd/adapter.cpp

```diff
@@ -2400,9 +2407,12 @@ HRESULT WinMDAdapter::TranslateWinMDAttributeUsageAttribute(mdTypeDef tkTypeDefO
     {
         IfFailGo(COR_E_BADIMAGEFORMAT);
     }
-    DWORD wfTargetValue = *(DWORD*)(pbWFUsageBlob + 2);
-    *pClrTargetValue = ConvertToClrAttributeTarget(wfTargetValue);
 
+    { // limiting the scope of wfTargetValue
```

Comment:
Can you please remove this comment? It is obvious and unnecessary for understanding the code.